docs/public-identity-infra/standard.md

```diff
@@ -0,0 +1,117 @@
+
+#### 3.1.4. Other fields
+
+The certificate MUST NOT contain other subject fields.
+
+### 3.2. Issuer Fields
+
+The Certificate Authorities MUST make sure to build an [RFC 5280](https://www.rfc-editor.org/rfc/rfc5280) compliant, unique, distinguished name. The exact fields used are left unspecified by this document.
+
+### 3.3. Certificate Uniqueness
+
+There MUST only be a single valid certificate with the same Distinguished Name at the same time. Certificates with the same Common Name MUST belong to the same physical person.
+
+### 3.4. Key Algorithms
+
+Certificates must use the RSA algorithm of minimum length 2048 or the ECDSA algorithm of minimum length 256.
+
+## 4. Issuing Policies
+
+Certificate issuing MUST be done after manual review by a human. The subject is responsible for proving they are legitimate to obtain the certificate, through official documents or online presence, when applicable. The Certificate Authority is responsible for verifying the uniqueness of the certificate delivered, across all commonly trusted authorities.
+
+### 4.1. Legal documents
+
+A subject requesting a certificate with their full name MAY submit legal documents proving who they are.
+
+### 4.2. Paid resources
+
+A subject requesting a certificate with their username MAY submit various paid resources proving who they are. 
+
+This includes, but not limited to:
+- Minecraft accounts
+- Internet domains
+
+### 4.3. Social resources
+
+A subject requesting a certificate with their username MAY submit various social, free, resources proving who they are.
+
+This includes, but not limited to:
+- Social media accounts: Discord, Reddit, Twitter, ...
+- Email addresses
+
+### 4.4. Issuing Thresholds
+
+For a certificate to be issued, the subject MUST submit one of the following:
+
+- One or more legal documents
+- Two or more paid resources
+- One paid resources and one or more social resources
+- Three or more social resources
+
+If a certificate with the same Common Name already exists, the subject MUST prove ownership of one of those certificates.
+
+Certificate Authorities MAY add additional requirements.
+
+### 4.5. Data retention
+
+Upon deliverance, the Certificate Authority MUST NOT keep any provided resources or documents in clear text format. The Certificate Authority MUST keep a hash of a unique identifier of the resources or documents and share that hash with other Certificate Authorities through a public database (format TBD).
+
+### 4.6. Data privacy
+
+The Certificate Authority MUST NOT make any information other than subject fields explicitly provided by the subject public. The Certificate Authority MUST adhere to the best privacy practices according to the European Union General Data Protection Regulations.
+
+### 4.7. Certificate Transparency
+
+The Certificate Authority MUST publish the issued certificate to three or more Certificate Transparency logs. 
```

Comment:
Out of curiosity: This implies that three or more of these logs would exist - so where would they be? Who would be responsible for maintaining them? Do they already exist, or does the MMPA need to set them up?